modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/display/context/factory/AICreatorOpenAIMenuItemFactoryImpl.java

```diff
@@ -91,6 +91,27 @@ public MenuItem createAICreatorCreateImageMenuItem(
 						LiferayWindowState.POP_UP
 					).buildString();
 				}
+			).put(
+				"isAICreatorOpenAIAPIKey",
+				() -> {
+					try {
+						if (Validator.isNotNull(
+								_aiCreatorOpenAIConfigurationManager.
+									getAICreatorOpenAIGroupAPIKey(
+										themeDisplay.getCompanyId(),
+										themeDisplay.getScopeGroupId()))) {
+
+							return true;
+						}
+					}
+					catch (ConfigurationException configurationException) {
+						if (_log.isDebugEnabled()) {
+							_log.debug(configurationException);
+						}
+					}
+
+					return false;
```

Comment:
This looks a bit weird. Why not keep the old "aiCreatorDALLEEnabled" attribute, and make it simply be `isEnabled() && apiKey() != null`?

Reply:
I've follow the same pattern as: 

https://github.com/liferay-lima/liferay-portal/blob/6f29802034e02288cc7bf1ce59e912d7cf715d14/modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java#L45

and 

https://github.com/liferay-lima/liferay-portal/blob/6f29802034e02288cc7bf1ce59e912d7cf715d14/modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java#L75

Reply:
Ok, sounds good then.